cpp/src/io/json/nested_json_gpu.cu

```diff
@@ -1583,6 +1583,9 @@ std::pair<rmm::device_uvector<PdaTokenT>, rmm::device_uvector<SymbolOffsetT>> ge
                                         thrust::make_discard_iterator(),
                                         fix_stack_of_excess_chars::start_state,
                                         stream);
+
+    // Make sure memory of the FST's lookup tables isn't freed before the FST completes
```

Comment:
I assume this is about `fix_stack_of_excess_chars`. Which part of it is not destroyed in stream order (AFAICT, must be a host-side buffer?)?

Reply:
@shrshi answered this offline :)